doc/developer/setup/index.md

```diff
@@ -172,6 +172,26 @@ See [AWS S3 Bucket Storage Setup](https://docs.aws.amazon.com/AmazonS3/latest/us
 
 There is no other configuration necessary in IRIDA to use cloud based storage. After adding these values to the configuration file you should be able to start up IRIDA, and it will use the cloud based storage that is configured.
 
+There isn't a local storage emulator available from Amazon AWS but we can make use of localstack to develop locally. The best way to make use of localstack is to use it with a docker container.
+
+1) Install `pip`, then use `pip install MODULE_NAME` to install `docker`, `awscli`, and `awscli-local`
```

Comment:
`docker` is not installed from `pip` can you may split this into 2 steps. 1 to install docker and 2 to install aws tools from pip

Reply:
Whoops! Updated in e76b3c528b347069ee6792f4eb67a62aad521a7d